apps/yjs/src/yjs-runtime.ts

```diff
@@ -78,7 +77,6 @@ export async function setupRuntimeSocket({
             type: `${type}_${content.name}`,
             text: content.text,
           });
-          resultMap.set(podId, oldresult);
```

Comment:
Is this unnecessary?

Reply:
It is necessary. Otherwise the stream won't be displayed until the very end. Try this:

```py
import time
for i in range(5):
    print(i)
    time.sleep(1)
```

Reply:
I noticed the yjs sync `resultMap` at least 3 times during a single `run`, and intended to reduce the frequency. But from the example above, I agree that yjs should honer the kernel execution behavior even though the sync cost is high. Let me revert the change.